test-runner/src/main/scala/scala/scalanative/testinterface/ComRunner.scala

```diff
@@ -35,7 +35,7 @@ class ComRunner(bin: File,
   private[this] var serverSocket: ServerSocket = _
   private[this] val socket =
     try {
-      serverSocket = new ServerSocket(0)
+      serverSocket = new ServerSocket(0, 1)
```

Comment:
Consider using comment-named params like this:
```suggestion
      serverSocket = new ServerSocket(/* port = */ 0, /* backlog = */ 1)
```

Reply:
Done. Good approach! Thank you.  I will have to add that idiom to my tool bag.
Scalafmt instisted on a space after the open-paren.